debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -767,7 +767,8 @@ else if (inconsistentSchemaHandlingMode == EventProcessingFailureHandlingMode.WA
             }
         }
         else {
-            logger.debug("Skipping {} event: {} for non-monitored table {}", typeToLog, event, tableId);
+            logger.debug("Filtering {} event: {} for non-monitored table {}", typeToLog, event, tableId);
+            metrics.onFilteredEvent(typeToLog);
```

Comment:
Unsure what the appropriate thing to pass into `onFilteredEvent()` is -- it's not referenced in the method.

Reply:
Please use the same string as is for the metrics code - `ventListener.onSkippedEvent("source = " + dataCollectionId);` - here it will be `tableId`

Reply:
What if `tableId` is null?  See the if statement on line 721